snek.yaml

```diff
@@ -0,0 +1,17 @@
+manifestVersion: subsquid.io/v0.1
+name: sneck
```

Comment:
`sneck`?

Reply:
Yes since we are using new deployment manifest I need to have different names for each indexer

I was thinking 
- `sneck` fror snek
- `basilick` for bsx